Networking/Networking/Model/Product/Product.swift

```diff
@@ -521,6 +525,9 @@ public struct Product: Codable, GeneratedCopiable, Equatable, GeneratedFakeable
 
         let menuOrder = try container.decode(Int.self, forKey: .menuOrder)
 
+        // Filter out metadata if the key is prefixed with an underscore (internal meta keys)
+        let customFields = (try? container.decode([MetaData].self, forKey: .metadata).filter({ !$0.key.hasPrefix("_")})) ?? []
```

Comment:
I think we should add unit tests to ensure that `customFields` are mapped as expected. Probably to `ProductMapperTests`. What do you think?

Reply:
Good point. I implemented this test: ff9b457560f3ea7ff728b7b8ce0ac01115da2cd6